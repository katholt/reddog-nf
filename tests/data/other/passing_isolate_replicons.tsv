isolate_23	contig_1
isolate_1	contig_1
isolate_2	contig_1
isolate_18	contig_1
isolate_22	contig_1
isolate_37	contig_1
isolate_32	contig_1
isolate_20	contig_1
isolate_30	contig_1
isolate_25	contig_1
isolate_29	contig_1
isolate_21	contig_1
isolate_28	contig_1
isolate_27	contig_1
isolate_19	contig_1
isolate_24	contig_1
isolate_36	contig_1
isolate_33	contig_1
isolate_26	contig_1
isolate_35	contig_1
isolate_34	contig_1
isolate_31	contig_1
isolate_1	contig_2
isolate_2	contig_2
