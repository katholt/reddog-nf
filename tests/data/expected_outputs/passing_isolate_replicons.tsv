../other/passing_isolate_replicons.tsv